script/Arcueid/Day 10/10_00_ARC10_7B.txt

```diff
@@ -1,27 +1,32 @@
 [sha:01d636b7bd7bd14fae9d4a5963ae8849a46f7376]{
 -- Page 10022, Offset 26338.
 -- 夜の街を闇雲に走る。
-I run in the dark through the city.
+// 夜 dropped
+I run at random through the city.
 }
 [sha:7a36e17dedac4144b31f17ea2ef20d83e971a4d7]{
 -- Page 10022, Offset 26339.
 -- アルクェイドの姿は見当たらない。
-I can't find Arcueid anywhere.
+I can't spot Arcueid anywhere.
 }
 [sha:71dfe432f9690fbdea03c738861b76a28ad5f1d4]{
 -- Page 10022, Offset 26340.
 -- 街は広すぎて、何の手がかりもなしで人間ひとりを捜し当てるのには時間が足りない。
-The city is too big. There isn't enough time to locate her myself, running around aimlessly like this.
+The city is too big, and I have no clues to go off.
+ It would take me forever to find her at this rate.
```

Comment:
i think the previous version was more correct. It's more like "I don't have time for an exhaustive search" instead of "it will take me too long". pretty minor difference either way